packages/okta-vue/src/Auth.spec.js

```diff
@@ -11,17 +11,28 @@ AuthJS.mockImplementation(() => {
   return mockAuthJsInstance
 })
 
+const localVue = createLocalVue()
+
 describe('Auth', () => {
   test('is a Vue plugin', () => {
     expect(Auth.install).toBeTruthy()
   })
   test('sets the right user agent on AuthJS', () => {
     const expectedUserAgent = `${pkg.name}/${pkg.version} foo`
-    createLocalVue().use(Auth, {
+    localVue.use(Auth, {
       issuer: '1',
       client_id: '2',
       redirect_uri: '3'
     })
     expect(mockAuthJsInstance.userAgent).toMatch(expectedUserAgent)
   })
+  test('sets the context for the $auth plugin', () => {
+    localVue.use(Auth, {
+      issuer: '1',
+      client_id: '2',
+      redirect_uri: '3',
+      response_type: 'a b'
+    })
+    expect(localVue.prototype.$auth).toBeDefined()
```

Comment:
We don't expose the configuration or the `AuthJS` object in Vue, so there is no good way to check this feat.

Instead, we're ensuring the object still will be created.

Reply:
Using mocking you can still test the desired side effect: that `oktaAuth.token.getWithRedirect` is called with the right values, given the input options given to the Auth plugin constructor.  Since there is already a mock of the internal authJs instance in this test file, you can use `jest.fn()` to add a mock version of `token.getWithRedirect()` to it and then use the [`mock.calls` property](https://facebook.github.io/jest/docs/en/mock-function-api.html#mockfnmockcalls) to see that it was called with the expected response_type